### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by sougou**
RateLimiter is for throttling SASL handshakes, which will come later; I lost workstation connectivity.

### CODE REVIEW

File: go/vt/client/client.go [hunk 1]
```diff
@@ -171,7 +171,7 @@ func makeBindVars(args []driver.Value) map[string]interface{} {
 	}
 	bv := make(map[string]interface{}, len(args))
 	for i, v := range args {
-		bv[fmt.Sprintf("v%d", i)] = v
+		bv[fmt.Sprintf("v%d", i+1)] = v
```
Comment (alainjobart) [inline go/vt/client/client.go]: How about a unit test?

**Comment by alainjobart** [comment]
LGTM after unit test :)

File: go/vt/client/client.go (hunk 1 above)
Comment (sougou) [inline go/vt/client/client.go]: There is a unit test already, which uses fakeserver_test.go. It was faulty.

Comment (alainjobart) [inline go/vt/client/client.go]: I missed the test file, my bad.

**Pull request merged**